contracts/addresses/11155111.json

```diff
@@ -88,10 +88,15 @@
       "EPOCH_VOTING_CUTOFF": "172800"
     },
     "governance": "0x7eedda08119826757c98a4680e94f3b5e1f33af6",
-    "uniV4DonationsInitiative": "0xc2348442f31d0182d3178d7b0c34ad58ddcf1400",
-    "curveV2GaugeRewardsInitiative": "0x0000000000000000000000000000000000000000",
-    "curvePool": "0x5b3f6b88fa29e85778394992d9bcd95ef2fceebe",
-    "gauge": "0x0000000000000000000000000000000000000000",
-    "LQTYToken": "0x878fc269a85619da792b054e2288cf5f9cc3e2e1"
```

Comment:
@bpierre heads up, I am renaming some fields in the manifest, because they're ambiguous: we'll have multiple Curve pools, gauges and initiatives on mainnet:
- `curveV2GaugeRewardsInitiative` => `curveUsdcBoldInitiative`
- `curvePool` => `curveUsdcBoldPool`
- `gauge` => `curveUsdcBoldGauge`

Also deleting `uniV4DonationsInitiative` as we're not going to deploy a Uni v4 initiative.